language/e2e-tests/src/tests/transaction_builder.rs

```diff
@@ -624,14 +565,47 @@ fn dual_attestation_payment() {
     }
 }
 
+fn create_dual_attestation_payment(
+    sender_address: AccountAddress,
+    receiver_address: AccountAddress,
+    amount: u64,
+    coin_type: TypeTag,
+    ref_id: Vec<u8>,
+    receiver_compliance_private_key: &Ed25519PrivateKey,
+) -> Script {
+    // UTF8-encoded string "@@$$LIBRA_ATTEST$$@@" without length prefix
```

Comment:
Please change this comment to say "ASCII-encoded". (I thought I had removed all the references to UTF-8 earlier, but I must have missed some or they were added back.) Or, better yet, maybe you could write the domain_separator as a byte string literal? e.g., b"@@$$LIBRA_ATTEST$$@@".to_vec()
Then there would be no need for the comment at all.